src/components/WorkPoolQueuesTable.vue

```diff
@@ -1,20 +1,18 @@
 <template>
   <p-content class="work-pool-queues-table">
     <p-list-header sticky>
-      <template v-if="selected">
-        <ResultsCount v-if="selected.length == 0" label="Work Queue" :count="filteredWorkPoolQueues.length" />
-        <SelectedCount v-else :count="selected.length" />
-        <p-button v-if="can.create.work_queue && !selected.length" small icon="PlusIcon" :to="routes.workPoolQueueCreate(workPoolName)" />
+      <ResultsCount v-if="selected.length == 0" label="Work Queue" :count="filteredWorkPoolQueues.length" />
```

Comment:
This ResultsCount (i.e. `3 Work Queues`) along with the Plus button to create a new work queue were previously hidden entirely if `can.delete.work_queue === false`.